### PR TITLE
on canary build, version should show "0.0.0-dev"

### PR DESCRIPTION
**Pull request by itscaro**

### CODE REVIEW

**Comment by itscaro** [comment]
@yxxhero not due to the changes :(

**Comment by itscaro** [comment]
ah ha found it

```
in test/integration/happypath.yaml: unable to update deps: unable to update 1 deps: Malformed version: dev
```

**Comment by yxxhero** [comment]
> Malformed version:

`dev` is not Semantic Versioning.

**Comment by yxxhero** [comment]
maybe 0.0.0-dev? @itscaro

**Comment by yxxhero** [comment]
@itscaro add unittests. Thanks very much.

**Comment by yxxhero** [comment]
@itscaro ping

File: pkg/app/version/version.go [hunk 1]
```diff
@@ -1,3 +1,24 @@
 package version
 
+import "fmt"
+
 var Version string
```
Comment (yxxhero) [inline pkg/app/version/version.go:5]: please add unittest for this file? Thanks very much.

**Comment by yxxhero** [comment]
@itscaro ping

**Comment by yxxhero** [comment]
@itscaro

**Comment by itscaro** [comment]
@yxxhero I will try to implement unit test this week

**Comment by itscaro** [comment]
@yxxhero could you have a look? thx

**Comment by yxxhero** [comment]
LGTM

**Pull request merged**